contents/docs/feature-flags/snippets/groups-flags-node.mdx

```diff
@@ -3,7 +3,7 @@ Update [posthog-node](https://posthog.com/docs/integrate/server/node) to version
 To check a flag that's matching by companies, specify groups in relevant feature flag call:
 
 ```node
-const isFlagEnabled = await posthog.isFeatureEnabled('new-groups-feature', '[distinct id]', false, { company: 'id:5' })
+const isFlagEnabled = await posthog.isFeatureEnabled('new-groups-feature', '[user distinct id]', false, { company: 'id:5' })
```

Comment:
Up to you, but `{ company: 'id:5' }` could be a little confusing. We could use `{ 'company': 'company_id_in_your_db' }`

Reply:
good catch!